components/form/appearance.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Appearance
+page_title: Form Appearance
+description: Appearance settings of the Form for Blazor.
+slug: form-appearance
+tags: telerik,blazor,form,appearance
+published: True
+position: 35
+---
+
+# Appearance Settings
+
+You can control the appearance of the form by setting the [Size](#size) attribute.
+
+## Size
+
+>The `Size` configuration of the Form affects the padding of all the editors and buttons.
```

Comment:
Move that not after the first paragraph to provide the main information for the parameter first and then list the specifics.